packages/@ember/-internals/views/lib/views/core_view.ts

```diff
@@ -45,14 +40,32 @@ const CoreView = (FrameworkObject.extend(Evented, ActionHandler, {
     @default null
     @private
   */
-  parentView: null,
+  declare parentView: View | null;
+
+  init(properties: object | undefined) {
```

Comment:
This whole pattern seems like it's in the "this looks like it should work but it makes me *super* nervous" bucket. Mind at least leaving a detailed comment in the places you've done it explaining the method so that when we come along to work on it (hopefully including just removing a lot of it!) 🔜 it'll be easier to follow?